actors/runtime/pom.xml

```diff
@@ -122,12 +122,12 @@ THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
         <dependency>
             <groupId>com.esotericsoftware</groupId>
             <artifactId>kryo</artifactId>
-            <version>3.0.3</version>
+            <version>4.0.0</version>
```

Comment:
Yes, I spotted that, but the only data that potentially lives between a deployment is actor directory and we never guarantee compatibility.